core/src/main/java/hudson/Functions.java

```diff
@@ -665,7 +667,25 @@ public static boolean isAutoRefresh(HttpServletRequest request) {
     public static boolean isCollapsed(String paneId) {
     	return PaneStatusProperties.forCurrentUser().isCollapsed(paneId);
     }
-    
+
+    public static boolean useUserTimeZone() {
```

Comment:
Wouldn't something like `isUserTimeZoneOverride` be more in line with the usual naming convention?